modules/local/querynator/cgiapi.nf

```diff
@@ -36,14 +37,17 @@ process QUERYNATOR_CGIAPI {
     def mutations_file = mutations ? "--mutations ${mutations}" : ""
     def translocation_file = translocations ? "--translocations ${translocations}" : ''
     def cnas_file = cnas ? "--cnas ${cnas}" : ''
+    def cancer = cancer ? cancer : 'Any cancer type'        // default to any cancer type if not specified
 
     """
+    export MPLCONFIGDIR=${workDir}/.config/matplotlib
+
     querynator query-api-cgi \\
         $mutations_file \\
         $translocation_file \\
         $cnas_file \\
         --outdir ${prefix}_cgi \\
-        --cancer $cancer \\
+        --cancer '$cancer' \\
         --genome $genome \\
```

Comment:
why are there '$cancer '  but $genome? Does this not run into troubles because ''Any cancer type'' is then handed?

Reply:
I don't think I undestood your question. Is it about the quotes (') ?

Reply:
ah ok. no that works fine